javascript/tagAutocomplete.js

```diff
@@ -831,6 +858,15 @@ function addResultsToList(textArea, results, tagword, resetList) {
             flexDiv.title = "✨ Frequent tag. Ctrl/Cmd + click to reset usage count."
         }
 
+        // Add 🔁 to indicate if has existed tag
+        if(IS_DAN_OR_E621_TAG_FILE && tagCount[result.text] >= 1) {
+            const textNode = flexDiv.querySelector(".acMetaText")
+            const span = document.createElement("span")
+            textNode.insertBefore(span, textNode.firstChild)
+            span.classList.add("existed")
+            span.title = `🔁 This tag has existed in before prompt.`
```

Comment:
This would be better grammatically, also no need for ` as it's not interpolated
```suggestion
            span.title = "🔁 The prompt already contains this tag";
```